shared/actions/chat.js

```diff
@@ -396,13 +396,13 @@ function _conversationLocalToFinalized (convo: ?ChatTypes.ConversationLocal): Fi
 
 function _inboxToConversations (inbox: GetInboxLocalRes, author: ?string, following: {[key: string]: boolean}, metaData: MetaData): List<InboxState> {
   return List((inbox.conversationsUnverified || []).map(convoUnverified => {
-    const msgBoxed = convoUnverified.maxMsgs && convoUnverified.maxMsgs.length && convoUnverified.maxMsgs[0]
+    const msgMax = convoUnverified.maxMsgSummaries && convoUnverified.maxMsgSummaries.length && convoUnverified.maxMsgSummaries[0]
```

Comment:
This looks fine (logically and passes flow). I'd have to setup gregor locally to actually test this but i think this'll either work or not in your branch so i'm confident it'll be fine 👍